source/jormungandr/tests/departure_board_tests.py

```diff
@@ -1104,3 +1103,107 @@ def test_routes_schedule_with_invalid_forbidden_uri(self):
+
+        Datetimes shown in the figure are all local datetimes
+         """
+
+        if not app.config['USE_SERPY']:
+            return
+
+        """              20170102
+                               from_datetime 20170102T1630       
+                                    |  
+                                    v
+         (Open)                             (Close)
+        08:00             15:00             26:45  
+         X_s1 ------------ X_S2 ------------ X_S3
+         
+        """
+        response = self.query_region("stop_points/X_S2/stop_schedules?from_datetime=20170102T1630"
+                                     "&data_freshness=base_schedule")
+
+        stop_schedules = response['stop_schedules']
+        assert stop_schedules
+        assert stop_schedules[0]['first_datetime']['date_time'] == "20171002T080000"
+        assert stop_schedules[0]['last_datetime']['date_time'] == "20171003T023000"
+
+        """              20170102
+                               from_datetime 20170103T0000      
+                                          |  
+                                          v
+        (Open)                             (Close)
+        08:00             15:00             26:45  
+         X_s1 ------------ X_S2 ------------ X_S3
+
+        """
+        response = self.query_region("stop_points/X_S2/stop_schedules?from_datetime=20170103T000000"
+                                     "&data_freshness=base_schedule")
+
+        stop_schedules = response['stop_schedules']
+        assert stop_schedules
+        assert stop_schedules[0]['first_datetime']['date_time'] == "20171002T080000"
+        assert stop_schedules[0]['last_datetime']['date_time'] == "20171003T023000"
+
+        """              20170102                                                20170103
+                                              from_datetime 20170103T0300       
+                                                         |  
+                                                         v
+        (Open)                             (Close)              (Open)                             (Close)
+        08:00             15:00             26:45      03:00    08:00             15:00             26:45  
+         X_s1 ------------ X_S2 ------------ X_S3                X_s1 ------------ X_S2 ------------ X_S3
+
+        """
+        response = self.query_region("stop_points/X_S2/stop_schedules?from_datetime=20170103T0300"
+                                     "&data_freshness=base_schedule")
+
+        stop_schedules = response['stop_schedules']
+        assert stop_schedules
+        assert stop_schedules[0]['first_datetime']['date_time'] == "20171003T080000"
+        assert stop_schedules[0]['last_datetime']['date_time'] == "20171003T023000"
+
+        """ 
+                        20170103                                                  20170104    
+                                             from_datetime 20170103T0300        
```

Comment:
Here, from the rest of your comment, it's the 4th you are testing, not the 3rd of january.